audit_report.py

```diff
@@ -1,11 +1,49 @@
 from course_finder import get_courses
 
+letter_grade_points = {
+                    "A": 4.000,
+                    "A-": 3.670,
+                    "B+": 3.330,
+                    "B": 3.000,
+                    "B-": 2.670,
+                    "C+": 2.330,
+                    "C": 2.000,
+                    "F": 0.000,
+                    "I": 0.000,
+                    "P": 0.000
+                }
 
 class AuditReport:
     def __init__(self, path_to_transcript, track):
         courses = get_courses(path_to_transcript)
         self.courses = courses
         self.track = track
+        
+    def get_gpas_section(self):
+        gpas_section = ""
+        gpas_section += self.get_core_gpa_section() + "\n"
+        gpas_section += self.get_elective_gpa_section() + "\n"
+        gpas_section += self.get_combined_gpa_section() + "\n"
+        return gpas_section
+        
+    def get_core_gpa_section(self):
+	    core_gpa_section = "Core GPA: "
+	    core_gpa = str(self.get_core_gpa)
```

Comment:
The error that Lisa is getting is occurring bc line 31 should be 
```
core_gpa = str(self.get_core_gpa())
```
Fix these everywhere